models/articles.models.js

```diff
@@ -45,3 +45,37 @@ exports.selectCommentsByArticleId = (article_id) => {
             return result.rows;
         });
 };
+
+exports.updateArticleVotes = (article_id, inc_votes) => {
+    if (!article_id || !inc_votes) {
+        return Promise.reject({
+            status: 400,
+            msg: 'missing required field!',
+        });
+    }
+    if (isNaN(inc_votes)) {
```

Comment:
You shouldn't need to check this, PSQL will throw it's own error if inc_votes is not a number. We stress TDD for this reason, you might find this test would have passed straight away without adding any code